HLTriggerOffline/Common/python/HLTValidation_cff.py

```diff
@@ -31,11 +32,13 @@
     +hltMultiPVValidation
     +egammaSelectors
     +ExoticaValidationProdSeq
+    +hltGsfTracksPreValidation
     )
 from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
 
 hltvalidation = cms.Sequence(
     HLTMuonVal
+    +hltMultiTrackValidationGsfTracks
```

Comment:
I'd prefer to have all MTV instances in the "prevalidation" Path (i.e. `hltassociation` sequence) for consistency (and because there is no reason for these to be in EndPath).